tests/save_load.test.js

```diff
@@ -109,21 +125,22 @@ test('Save saveAsJSON should return error message if error is encountered', asyn
     removeListener.mockImplementation(() => resolve());
   });
 
-  await save.saveAsJSON([{ prop1: 'log1' }]).catch((error) => {
-    expect(error.message).toBe('error');
+  const log = [{ prop1: 'log1' }];
+
+  await save.saveAsJSON(log).catch((error) => {
+    expect(error.message).toBe('saveerror');
   });
 
   Promise.all([promiseRevokedURL, promiseRemovedListener]);
 
-  expect(download).toHaveBeenCalledTimes(1);
+  expect(createObjectURL).toHaveBeenCalledWith(new Blob([JSON.stringify(log)]));
```

Comment:
This check doesn't work as expected for the reason that I provided in https://github.com/mozilla/extension-activity-monitor/pull/2/files/168cec3ae3c1182806ec14f4604ec946dbeffa91..6ac6cf9062aad9f46792ce85bca8f376d3c271ee#r440697786